.gitignore

```diff
@@ -126,9 +126,6 @@ $tf/
 # Guidance Automation Toolkit
 *.gpState
 
-# Visual Studio Code
-.vscode
-
```

Comment:
This ensured that the only `.vscode` files in the repository are `extensions.json` and `settings.json`. So, this `.gitignore` is intentional.

Reply:
It's normal / intended / best practice to be checked into version control - it's project specific settings only, any unwanted changes to the .vscode in code review.